CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.16.0]
+### Changed
+- **BREAKING:** Refactor code to split keyring from bridge logic. ([#156](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/156))
```

Comment:
Nit: This change entry could be improved (see [the suggestions left here](https://github.com/MetaMask/eth-trezor-keyring/pull/177#discussion_r1231110025) for example)

Reply:
Makes sense, thanks.